City-of-Helsinki/drupal-helfi-etusivu
# set the baseurl to fix the sitemap paging

# [UHF-10812](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10812)

Sitemap paging leads back to sitemap index instead of the actual sitemap.

## What was done
Override the sitemap entity class and the toUrl method.


## How to install
* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10812`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Run `drush simple-sitemap:generate`
- Go to see the paged sitemap: https://helfi-etusivu.docker.so/fi/sitemap.xml
  - You should see urls with a page parameter
    - The link should contain a langcode  
- Open one of the paged link
  - The links should lead to a proper sitemap with correct links (no duplicate langcode on url or other weirdness)



[UHF-10812]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10812?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
